source/language/gates.rst

```diff
@@ -283,42 +285,93 @@ A gate modifier is a keyword that applies to a gate. A modifier
 on the same or larger Hilbert space. We include modifiers in OpenQASM
 both for programming convenience and compiler analysis.
 
-The modifier ``inv @`` replaces its gate argument :math:`U` with its inverse
-:math:`U^\dagger`. The inverse of any gate can be defined recursively by
-reversing the order of the gates in its definition and replacing each of
-those with their inverse. The base case is given by replacing ``inv @ CX`` with ``CX`` and
-``inv @ U(θ, ϕ, λ)`` by ``U(-θ, -λ, -ϕ)``.
+The modifier ``ctrl @`` replaces its gate argument :math:`U` by a
+controlled-:math:`U` gate. If the control bit is 0, nothing happens to the target bit.
+If the control bit is 1, :math:`U` acts on the target bit. Mathematically, the controlled-:math:`U`
+gate is defined as :math:`C_U = I \otimes U^c`, where :math:`c` is the integer value of the control
+bit and :math:`C_U` is the controlled-:math:`U` gate. The new control qubit is prepended to the
+argument list for the controlled-:math:`U` gate. The modified gate does not use any additional
+scratch space and may require compilation to be executed. A special case is the controlled phase
+gate, which is defined as :math:`ctrl @ gphase(a) = U(0,0,a)`.
 
 .. code-block:: c
 
-   gate rzm(theta) q1 {
-       inv @ rzp(theta) q1;
+   // Define a controlled Rz operation using the ctrl gate modifier.
+   // q1 is control, q2 is target
+   gate crz(θ) q1, q2 {
+       ctrl @ U(θ, 0, 0) q1, q2;
```

Comment:
The way this is written in `stdgates` is clearer, i.e. `ctrl @ rz(θ) q1, q2;`.

Reply:
Addressed in https://github.com/Qiskit/openqasm/pull/203/commits/4471686f9e8f72a249b267a99622e1a86150777b.